docs/website/docs/v1.x/dsl/request/index.md

```diff
@@ -1 +1,91 @@
-# Work in progress
\ No newline at end of file
+# Request
+ 
+**ZIO HTTP** `Request` is designed in the simplest way possible to, decode HTTP `Request` into a ZIO HTTP request.
+ It supports all HTTP request methods (as defined in [RFC2616](https://datatracker.ietf.org/doc/html/rfc2616) ) and headers along with custom methods and headers.
+ 
+## Creating a Request
+
+`Request` can be created with `method`, `url`, `headers`, `remoteAddress` and `data`. 
+Creating requests using `Request` is useful while writing unit tests.
+
+The below snippet creates a request with default params, `method` as `Method.GET`, `url` as `URL.root`, `headers` as `Headers.empty`, `data` as `HttpData.Empty`, `remoteAddress` as `None`
+```scala
+val request: Request = Request()
+```
+
+## Matching and Extracting Requests
+
+`Request` can be extracted into an HTTP Method and Path via the -> object. On the left side is the `Method`, and on the right side, the `Path`.
+
+```scala
+Method.GET -> !! / "text"
+```
+### Method
+ `Method` represents HTTP methods like POST, GET, PUT, PATCH, and DELETE.
+You can create existing HTTP methods such as `Method.GET`, `Method.POST` etc or create a custom one.
+ 
+
+### Path
+ `Path` can be created using
+  - `!!` which represents the root
+  - `/` which represents the path delimiter and starts the extraction from the left-hand side of the expression
+  - `/:` which represents the path delimiter and starts the extraction from the right-hand side of the expression and can match paths partially 
```

Comment:
Can there be a small example here explaining the term extraction from left or right side?

Reply:
I added just below this